Library/Formula/mplayer.rb

```diff
@@ -22,22 +24,9 @@ class Mplayer < Formula
     patch :DATA
   end
 
-  option "without-osd", "Build without OSD"
-
   depends_on "yasm" => :build
   depends_on "libcaca" => :optional
-  depends_on :x11 => :optional
-
-  deprecated_option "with-x" => "with-x11"
-
-  if build.with?("osd") || build.with?("x11")
-    # These are required for the OSD. We can get them from X11, or we can
-    # build our own.
-    depends_on "fontconfig"
-    depends_on "freetype"
-    depends_on "libpng"
-  end
```

Comment:
OSD is (as seen in my commit and pull request) not longer a reasonable option. I don't see --enable-menu in .configure but I do see --enable-gui. Please refer to the pull request comment for my reasoning of getting rid of it. I'll be happy to respond and answer any arising questions